### PR TITLE
Updated Deployment description for clarity.

### PR DESCRIPTION
**Pull request by brentcklein**
Rephrases part of the description of a Deployment in the "Deploy an App" section of the "Kubernetes Basics" tutorial to be more clear.

Fixes https://github.com/kubernetes/website/issues/19312

### CODE REVIEW

**Comment by brentcklein** [comment]
signed

**Comment by brentcklein** [comment]
signed

File: content/en/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro.html [hunk 1]
```diff
@@ -31,7 +31,8 @@ <h3>Kubernetes Deployments</h3>
                 Once you have a running Kubernetes cluster, you can deploy your containerized applications on top of it.
                 To do so, you create a Kubernetes <b>Deployment</b> configuration. The Deployment instructs Kubernetes
                 how to create and update instances of your application. Once you've created a Deployment, the Kubernetes
-                master schedules mentioned application instances onto individual Nodes in the cluster.
+                master schedules the application instances included in that Deployment to run on individual Nodes in the
```
Comment (rajeshdeshpande02) [inline content/en/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro.html]: Can you please make a minor change as below 
'master schedules the application instances specified in that Deployment to run on individual Nodes in the'

Comment (sftim) [reply content/en/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro.html]: @brentcklein, you could also change “master” to “control plane” if you're confident that helps learners understand things better.

Comment (brentcklein) [reply content/en/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro.html]: Will do. Should “control plane” be capitalized?

Comment (sftim) [reply content/en/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro.html]: > Should “control plane” be capitalized?

I think it's fine either way.

**Comment by sftim** [comment]
@brentcklein any update on this?

File: content/en/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro.html (hunk 1 above)
Comment (sftim) [reply content/en/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro.html:34]: @brentcklein any update? It'd be great to get those tweaks in if you can. If you're short on time, we can merge this and log a new issue to track the tweaking.

**Pull request merged**